platform/m480/inc/configurations.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2016-2017, Nuvoton, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __CONFIGURATIONS_H__
+#define __CONFIGURATIONS_H__
+
+/*******************************************************************************
+ * Family-wide configurations
+ ******************************************************************************/
+
+/* The symbols below *must* be calculated from values across the family. */
+
+/* Maximum number of vectors seen across the family:
+ *   NVIC_VECTORS = max(NVIC_VECTORS_i) for i in family */
+#define NVIC_VECTORS 96
+
+/* Minimum memory requirements:
+ *   FLASH_LENGTH_MIN = min(FLASH_LENGTH_i) for i in family
+ *   SRAM_LENGTH_MIN = min(SRAM_LENGTH_i) for i in family */
+#define FLASH_LENGTH_MIN 0x40000
+#define SRAM_LENGTH_MIN  0x20000
```

Comment:
@Patater 
> Does the M480-family target with the least amount of SRAM have 128 KiB of SRAM available continuously starting at address 0x20000000?

I fixed it to 96KiB. Please refer to https://github.com/ARMmbed/uvisor/pull/446/commits/4d977594fa50cab68c25e93236783d64d5d78d5d.

>Does the M480-family target with the least amount of flash have 256 KiB of flash available continuously starting at address 0x00000000?

Yes.